source/plugin/permissions.rst

```diff
@@ -2,9 +2,310 @@
+according to their needs using the default's persistent ``SubjectData``. If you would like to provide a configuration
+guideline for server owners use ``PermissionDescription's`` role-templates instead.
+
+.. warning::
+
+    You should think carefully before granting default permissions to users. By granting the permissions you are
+    assuming that all server owners will want these defaults (at least the first time the plugin runs) and that
+    exceptions will require server owners to explicitly deny the permissions (which can't even be done without a custom
+    permissions service implementation). This should roughly correspond to a guest on a single player lan world without
+    cheats. For example a chat plugin would allow sending chat messages by default to imitate vanilla game behaviour
+    for features that were changed by the plugin.
+
+.. note::
+
+    Its only specified for the default ``Subjects`` that the persistent ``SubjectData`` takes precedence over transient
+    ones.
+
+If neither the Subject, nor any of its parents, nor the defaults grant and deny a permission, then it is automatically
+denied.
 
 .. note::
 
-    This page has not been filled yet by our Documentation Team. If you feel like you can help, you can do so on `our
-    GitHub repository <https://github.com/spongepowered/spongedocs>`_. Also see the `related GitHub Issue
-    <https://github.com/SpongePowered/SpongeDocs/issues/32>`_ for more information on what is required.
+    Order of precedence in descending order:
+    
+    * Subject itself
+    * Parent Subjects
+    * SubjectCollection Defaults
+        * Persistent
+        * Transient
+    * PermissionService Defaults
+        * Persistent
+        * Transient
+    * Deny permission
+
+SubjectCollections
+==================
+
+A container for subjects that can be used to obtain a Subject by name.
+These are the default Subject Collections:
+
+* User
+    * Contains all on-line ``Players`` and all off-line ``Users`` (at least those with none-default settings).
+* Group 
+    * Contains all group ``Subject``. Groups are a simple way of structuring a ``Subject's`` inheritance tree using
+      named ``Subjects``. Groups should be used if a specific subset of ``Subjects`` have additional permission
+      settings such as a team, faction or role.
+* System
+    * Contains other ``Subjects`` used by the server such as the the console and possible remote consoles. 
+* Command Block
+    * Contains all ``Subjects`` for command blocks. These are useful if you would like to run a ``CommandBlock`` only
+      with the permissions of the creator.
+* Role Template
+    * Contains all role template subjects that are used in ``PermissionDescriptions``. Useful to lookup all recommended
+      permissions for a user. These should not be used for inheritance.
+
+.. note::
+
+    If ``SubjectCollections are queried for a ``Subject`` it will automatically be created, if it does not already
```

Comment:
`If` --> `When`
`it will automatically be created, if it does not` --> `they will automatically be created, if they do not`